reading.py

```diff
@@ -0,0 +1,38 @@
+from gmsh_io import GmshIO
+from operator import add
+import numpy as np
+from correlated_field import SpatialCorrelatedField
+import matplotlib.pyplot as plt
+
+# Read the mesh network of the model:
+gio = GmshIO()
+with open('C:\\Users\\Clara\\Documents\\Intec\\MLMC_Python\\rectangle.msh') as f:
+    gio.read(f)
+   
+# Getting the centers for each element:
+coord = np.zeros((len(gio.elements),3))
+for i in range(len(gio.elements)):
+    one_el = gio.elements[i+1]
+    index = one_el[2]
+    coord[i] = map(add,gio.nodes[index[0]],map(add,gio.nodes[index[1]],gio.nodes[index[2]]))
+coord = coord/3
+
+# Generating the "field" (conductivity)
+pole = SpatialCorrelatedField(corr_exp = 'gauss', dim = 3, corr_length = 2.6,aniso_correlation = None,  )
+pole.set_points(coord, mu = 3, sigma = 1.0)
+log_K        = pole.sample()
+conductivity = log_K   
+
+# Plotting the conductivity field (optional)
+# plt.scatter(coord[:,0],coord[:,1],c = conductivity)
+# plt.colorbar()    
+
+with open('C:\\Users\\Clara\\Documents\\Intec\\MLMC_Python\\vodivost.msh',"wb") as fout:

```

Comment:
Tohle implementovat do gmsh_io:
- zapise se to do nejake te struktury pro data:
```
  element_values = np.array()
  fields = { field_name: element_values }
  element_data = { time: fields}
```
- a pak se zavola write_ascii, kde se to zformatuje

Reply:
Ta procedura self.write_ascii je pro zapsani meshe, zakladni vypoctovy site pro flow model, nikoliv pro msh. soubor kde se zadavaji pole/hodnoty. Vzhledem k tomu, ze tudle sit mam na vstupu potrebuju ji znovu zapisovat??

Reply:
Rozumim, asi bude lepsi udelat samostatnou metodu `write_data_ascii`. Pak bude mozne bud zapsat sit i data, nebo kazde zvlast.